app/src/main/java/org/simple/clinic/facility/FacilityPayload.kt

```diff
@@ -56,7 +56,11 @@ data class FacilityPayload(
     val updatedAt: Instant,
 
     @Json(name = "deleted_at")
-    val deletedAt: Instant?
+    val deletedAt: Instant?,
+
+    // TODO(vs): 2020-01-22 Make this non-nullable once the feature is in PROD
```

Comment:
As in? Once the change is available on PROD, we can make this non-nullable and remove the default fallback. Is this what you're referring to?

Reply:
I meant this is dependent on the back-end changes going live that's all.
Since the current API doesn't support it that is the reason for keeping it nullable, right?

Reply:
Yes.